CHANGELOG.rst

```diff
@@ -26,6 +26,9 @@ Minor changes
 -------------
 
 - Remove helper ``get_site_statistics()`` (#8705)
+- ``ckan db clean`` now automatically clears the search index to prevent orphaned entries (#8347)
+- ``ckan search-index rebuild`` now automatically clears orphaned packages after rebuilding (#8347)
+- Removed ``--clear`` option from ``ckan search-index rebuild`` command (#8347)
```

Comment:
Add these to a file in the changes directory instead and they will be pulled in to the changelog as part of the next release.